### PR TITLE
chore: bump driver version and update changelog

### PR DESCRIPTION
**Pull request by karenc-bq**
### Description
 - Bump versioning to v2.3.6
 - Update changelog

### By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,14 +3,23 @@ All notable changes to this project will be documented in this file.
...
 - Documentation:
   - Aurora Initial Connection Strategy Plugin. See [UsingTheAuroraInitialConnectionStrategyPlugin](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheAuroraInitialConnectionStrategyPlugin.md)
+  - Additional instructions to enable logging for Spring and Spring Boot. See [Logging](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/UsingTheJdbcDriver.md#logging).
+
+### :bug: Fixed
+- Connection identification and tracking in the host list provider (PR #943)[https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/943].
+- Green node endpoint replacement, allowing the AWS Advanced JDBC Driver to detect and connect to green nodes after Blue/Green switchover (PR# 948)(https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/948). Addresses [issue #678](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/678).
+- MariaDB Pool Datasource support. Addresses [issue #957](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/957)
```
Comment (crystall-bitquill) [inline CHANGELOG.md:17]:
```suggestion
- MariaDB Pool Datasource support. Addresses [issue #957](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/957).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,14 +3,23 @@ All notable changes to this project will be documented in this file.
...
 
 ### :magic_wand: Added
+- Okta Authentication Support. See [UsingTheOktaAuthPlugin](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheOktaAuthPlugin.md).
 - Documentation:
   - Aurora Initial Connection Strategy Plugin. See [UsingTheAuroraInitialConnectionStrategyPlugin](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheAuroraInitialConnectionStrategyPlugin.md)
+  - Additional instructions to enable logging for Spring and Spring Boot. See [Logging](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/UsingTheJdbcDriver.md#logging).
+
+### :bug: Fixed
+- Connection identification and tracking in the host list provider (PR #943)[https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/943].
+- Green node endpoint replacement, allowing the AWS Advanced JDBC Driver to detect and connect to green nodes after Blue/Green switchover (PR# 948)(https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/948). Addresses [issue #678](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/678).
```
Comment (crystall-bitquill) [inline CHANGELOG.md:16]:
```suggestion
- Green node endpoint replacement, allowing the AWS JDBC Driver to detect and connect to green nodes after Blue/Green switchover (PR# 948)(https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/948). Addresses [issue #678](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/678).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,14 +3,23 @@ All notable changes to this project will be documented in this file.
...
+- MariaDB Pool Datasource support. Addresses [issue #957](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/957)
 
 ### :crab: Changed
 - Log level of `Failover.startWriterFailover` and `Failover.establishedConnection` from `fine` to `info` for better visibility of failover-related logs ([Issue #890](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/890)).
+- Documentation:
+  - Telemetry's connection properties. See [Telemetry](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/Telemetry.md).
```
Comment (crystall-bitquill) [inline CHANGELOG.md:22]:
```suggestion
- Telemetry's connection property documentation. See [Telemetry](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/Telemetry.md).
```

File: Maintenance.md
Comment (crystall-bitquill) [inline Maintenance.md:1]: Could you please also update the release schedule table in this file?

File: README.md
Comment (karenc-bq) [inline README.md:1]: @hsuamz Could you please review this update

**Review by crystall-bitquill — APPROVED** [review]

File: README.md
Comment (hsuamz) [reply README.md:1]: Thanks, LGTM.

**Review by hsuamz — APPROVED** [review]

**Pull request merged**